test/org/sagebionetworks/bridge/services/StudyConsentServiceImplTest.java

```diff
@@ -51,20 +52,22 @@
 
     @Resource
     private StudyConsentService studyConsentService;
-
+    
     private Study study;
     
     @Before
     public void before() {
+        String id = TestUtils.randomName(StudyConsentServiceImplTest.class);
+        
         study = new DynamoStudy();
-        study.setIdentifier(STUDY_ID.getIdentifier());
-        study.setName("A test study");
-        study.setSponsorName("A sponsor name");
+        study.setIdentifier(id);
+        study.setName(id);
+        study.setSponsorName(id);
```

Comment:
Minor: We probably shouldn't stuff the id into the name and sponsorName, just to make sure our tests don't "cross the streams", so to speak.